wgsl/index.bs

```diff
@@ -559,39 +621,39 @@ type.
 
 ### Read-only Storage Texture Types ### {#texture-ro}
 <pre class='def'>
-`texture_ro_1d<image_storage_type>`
-  %1 = OpTypeImage type(image_storage_type) 1D 0 0 0 2 image_storage_type ReadOnly
+`texture_ro_1d<texel_format>`
+  %1 = OpTypeImage type(texel_format) 1D 0 0 0 2 texel_format ReadOnly
 
-`texture_ro_1d_array<image_storage_type>`
-  %1 = OpTypeImage type(image_storage_type) 1D 0 1 0 2 image_storage_type ReadOnly
+`texture_ro_1d_array<texel_format>`
+  %1 = OpTypeImage type(texel_format) 1D 0 1 0 2 texel_format ReadOnly
 
-`texture_ro_2d<image_storage_type>`
-  %1 = OpTypeImage type(image_storage_type) 0 0 0 2 image_storage_type ReadOnly
+`texture_ro_2d<texel_format>`
+  %1 = OpTypeImage type(texel_format) 0 0 0 2 texel_format ReadOnly
 
-`texture_ro_2d_array<image_storage_type>`
-  %1 = OpTypeImage type(image_storage_type) 0 1 0 2 image_storage_type ReadOnly
+`texture_ro_2d_array<texel_format>`
+  %1 = OpTypeImage type(texel_format) 0 1 0 2 texel_format ReadOnly
 
```

Comment:
Seems like this lacks 2D annotation right after `type(texel_format)` (all others do have)

Reply:
Fixed in dc7f7538a